src/Microsoft.ML.Data/Model/ModelSaveContext.cs

```diff
@@ -11,9 +11,9 @@ namespace Microsoft.ML.Runtime.Model
 {
     /// <summary>
     /// This is a convenience context object for saving models to a repository, for
-    /// implementors of ICanSaveModel. It is not mandated but designed to reduce the
+    /// implementors of <see cref="ICanSaveModel"/>. It is not mandated but designed to reduce the
     /// amount of boiler plate code. It can also be used when saving to a single stream,
-    /// for implementors of ICanSaveInBinaryFormat.
+    /// for implementors of <see cref="ICanSaveInBinaryFormat"/>.
     /// </summary>
     public sealed partial class ModelSaveContext : IDisposable
```

Comment:
This is somewhat disappointing. I wanted to actually internalize this class altogether. But, I can't, because of the interface `ICanSaveModel`, which must be a public interface due to the fact that `IDataTransform` *does* and `ITransformer` *should* descend from it, and the second at least needs co/contravariance. It may be that I will be able to internalize `ModelLoadContext` in the future. I already made the constructor on this thing public so people can't just go creating them willy nilly, but I should probably just internalize everything public here with a `BestFriend`.